lib/context-menu.coffee

```diff
@@ -21,6 +21,14 @@ module.exports = ->
           'command': 'git-plus-context:diff'
         },
         {
+          label: 'Git diff branches',
```

Comment:
The ones you've registered are the global commands bound to the atom-workspace element.

The tree view context menu commands need to be bound to the `.tree-view` selector. If you look at line 151, you'll see that the context menu commands are wired differently and are separate models. If you try to invoke your new commands from the tree-view, they won't work.